ProcessMaker/Console/Commands/Install.php

```diff
@@ -212,13 +214,26 @@ public function handle()
 
         // Set laravel echo server settings
         $this->env['LARAVEL_ECHO_SERVER_AUTH_HOST'] = $this->option('echo-host') ? $this->option('echo-host') : $this->env['APP_URL'];
-        
+
         // Set path and Docker settings
         Storage::disk('local')->makeDirectory('scripts');
         $this->env['PROCESSMAKER_SCRIPTS_HOME'] = storage_path('app/scripts');
         $this->env['DOCKER_HOST_URL'] = $this->env['APP_URL'];
         $this->env['HOME'] = base_path();
-        
+
+        // Check for docker installation path if not under Win OS and set docker env path var
+        if (strtoupper(substr(PHP_OS, 0, 3)) !== 'WIN') {
+            $dockerBinPath = system('which docker');
```

Comment:
I recommend using `exec` here instead of `system` so that it does not print the output on screen during installation.